tests/e2e/performance/test_pvc_multi_snapshot_performance.py

```diff
@@ -0,0 +1,156 @@
+)
+from ocs_ci.ocs.resources import pvc
+from tests import helpers
+
+log = logging.getLogger(__name__)
+
+
+@performance
+@skipif_ocp_version('<4.6')
+@skipif_ocs_version('<4.6')
+class TestPvcMultiSnapshotPerformance(E2ETest):
+    """
+    Tests to measure PVC snapshots creation performance & scale
+    The test is trying to to take the maximum number of snapshot for one PVC
+    """
+
+    def test_pvc_multiple_snapshot_performance(
+        self, interface_iterate, teardown_factory, storageclass_factory,
+        pvc_factory, pod_factory
+    ):
+        """
+        1. Creating PVC
+           size is depend on storage capacity, but not less then 1 GiB
+           it will use ~75% capacity of the Storage, Min storage capacity 1 TiB
+        2. Fill the PVC with 80% of data
+        3. Take a snapshot of the PVC and measure the time of creation.
+        4. re-write the data on the PVC
+        5. Take a snapshot of the PVC and measure the time of creation.
+        6. repeat steps 4-5 the numbers of snapshot we want to take : 512
+        7. print all information.
+
+        Raise:
+            StorageNotSufficientException: in case of not enough capacity
+
+        """
+        create_times = []  # list for all snapshots results
+        num_of_snaps = 512  # number of snapshots to take
+
+        # Getting the total Storage capacity
+        ceph_cluster = CephCluster()
+        ceph_capacity = int(ceph_cluster.get_ceph_capacity())
+
+        # Use 70% of the storage capacity in the test
+        capacity_2_use = int(ceph_capacity * 0.7)
+
+        # since we do not want to use more then 75%, we add 35% to the needed
+        # capacity, and minimum PVC size is 1 GiB
+        need_capacity = int((num_of_snaps + 2) * 1.35)
+        # Test will run only on system with enough capacity
+        if capacity_2_use < need_capacity:
+            err_msg = f'The system have only {ceph_capacity} GiB, '
+            err_msg += f'we want to use only {capacity_2_use} GiB, '
+            err_msg += f'and we need {need_capacity} GiB to run the test'
+            log.error(err_msg)
+            raise exceptions.StorageNotSufficientException(err_msg)
+
+        # Calculating the PVC size in GiB
+        pvc_size = int(capacity_2_use / (num_of_snaps + 2))
+
+        self.interface = interface_iterate
+        self.sc_obj = storageclass_factory(self.interface)
+
+        self.pvc_obj = pvc_factory(
+            interface=self.interface,
+            size=pvc_size,
+            status=constants.STATUS_BOUND
+        )
+
+        self.pod_obj = pod_factory(
+            interface=self.interface,
+            pvc=self.pvc_obj,
+            status=constants.STATUS_RUNNING
+        )
+
+        # Calculating the file size as 80% of the PVC size
+        filesize = self.pvc_obj.size * 0.80
+        # Change the file size to MB for the FIO function
+        file_size = f'{int(filesize * constants.GB2MB)}M'
```

Comment:
can we use https://github.com/red-hat-storage/ocs-ci/blob/9e04a842930d30c313fd51f6f28e1764a9512cde/ocs_ci/utility/utils.py#L2284 here instead?

Reply:
can not use since this function convert in multiple of 1000, and the constants.GB2MB is 1024